### PR TITLE
create team minutes for March 27

### LINKED ISSUE #295: Team Meeting minutes for March 27
**Describe what needs to get done**

- take minutes during the meeting

**Describe the solution you'd like**

- markdown file in docs>meetings>team

### PR DESCRIPTION
**Pull request by vnajack**
- closes #295

### CODE REVIEW

File: docs/meetings/team/2023_03_27.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+**Date** March 27, 2023
+
+## 👋 Attendence
+
+- Veronica
+- Edouard
+
+Attendance Notes:
+
+Matt and Logan were unable to attend today
```
Comment (LoganParker) [inline docs/meetings/team/2023_03_27.md:18]: Sorry!!!

File: docs/meetings/team/2023_03_27.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## 🃏 Joke of the Day
+
+I have a brand new whiteboard, it's quite remarkable.
```
Comment (LoganParker) [inline docs/meetings/team/2023_03_27.md:22]: This is a good one, im sad I missed hearing it in person

File: docs/meetings/team/2023_03_27.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- We were informed we won't receive anymore assets from the Kukarella artist
+
+## 📋 This Week
+
+Peer Testing II issues found
+
+- Blinking probability changes (Veronica)
+- Documentation for server-side AWS implementation (Matt & Ed)
+- Documentation links to tooling & standard error codes
+- Documentation for tooling (V added screenshots in the issue for Logan, but isn't sure which screenshots he wanted)
```
Comment (LoganParker) [inline docs/meetings/team/2023_03_27.md:49]: just of the layers!!

File: docs/meetings/team/2023_03_27.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- Documentation for how to add transition frames (Veronica)
+- Documentation to say where new character image assets should go
+
+Brainstorming for the promo video
+
+- we could have Barb or Boy speak the lines
```
Comment (LoganParker) [inline docs/meetings/team/2023_03_27.md:55]: I think this is a good idea, might require some extra video editing but I think we could overlay them on a screen recording and remove the white background too

**Review by LoganParker — APPROVED** [review]
Thanks for taking these!! Sorry I was unable to attend the meeting today

**Review by edouarde1 — APPROVED** [review]
Thanks for the minutes!

**Pull request merged**